test.rb

```diff
@@ -0,0 +1,43 @@
+require 'pry'
+
+class Animal
+  attr_accessor :name
+
+  def initialize (name)
+    @name = name
+  end
+
+  def greet
+    puts "Buenos Dias"
+  end
+
+end
+
+pumba =  Animal.new("Pumba")
+
+class Lion < Animal
+  attr_accessor :king
+  @@pack = 0
+  @@pack += 1
+
```

Comment:
missing the `initialize` method where you add `self` to the `pack` and set the attribute of `king`